gql/directive.go

```diff
@@ -150,9 +165,10 @@ func isNotLoggedIn(ctx context.Context, obj interface{}, next graphql.Resolver)
 	authorization := header.Get("Authorization")
 
 	if authorization != "" {
-		user := postgres.GetUserByToken(ctx, authorization)
+		payload, _ := util.VerifyUserToken(authorization)
+		userID := payload.Get("userID")
 
-		if user != nil {
+		if userID != "" {
```

Comment:
Just having the `userID` in the payload does not imply the user is logged in, the token could be revoked or non-existent in the database.